src/main/java/frc/trigon/robot/poseestimation/poseestimator/StandardDeviations.java

```diff
@@ -0,0 +1,51 @@
+package frc.trigon.robot.poseestimation.poseestimator;
+
+import edu.wpi.first.math.geometry.Transform2d;
+
+public class StandardDeviations {
+    private final double translation, theta;
+
+    /**
+     * Constructs an object that stores how ambiguous the estimated pose of the robot is.
+     * The greater the number, the less trustworthy the pose is.
+     *
+     * @param translation the ambiguity of the translation aspect of the pose estimation
+     * @param theta       the ambiguity of the rotation aspect of the pose estimation
+     */
+    public StandardDeviations(double translation, double theta) {
+        this.translation = translation;
+        this.theta = theta;
+    }
+
+    /**
+     * Combines this with another {@link StandardDeviations}.
+     *
+     * @param other the {@link StandardDeviations} to combine with
+     * @return the combined {@link StandardDeviations}
+     */
```

Comment:
This jdoc doesn't give me any more info than the function name 
Explain where this might be used, the limitations etc.